### DISCUSSION TITLE
UGS Start Fullscreen

### DISCUSSION THREAD

**Discussion by Robot-John** [winder/Universal-G-Code-Sender#2469] [Help]
Hi all,

I've done some digging to try to figure this out with no luck.

I have UGS running on a Raspberry Pi with a 7" LCD display. It's working out well and I'm just now handling the little details.

I have UGS opening automatically at startup and due to my small display size, I would like UGS to default to full screen mode when opening.

Is there an option to do this from a Linux shell command?

Something like `cnc@controller: sudo /home/cnc/Programs/UGS/bin/ugsplatform -full`

Thanks.

**Comment by breiler** [comment]
There is now an option to start UGS in fullscreen with the command line option `--fullscreen`. Available in the latest nightly build here: https://github.com/winder/Universal-G-Code-Sender?tab=readme-ov-file#downloads
